src/main/resources/com/cloudbees/plugins/credentials/ViewCredentialsAction/index.jelly

```diff
@@ -85,7 +85,7 @@
                    class='model-link inside'>
                   <j:choose>
                     <j:when test="${d.domain.global}">
-                      (global)
+                      ${%(global)}
```

Comment:
This syntax does not work. Nothing appears, since `(`…`)` is the syntax to introduce a parameterized bundle key.

You would need to create an `index.properties` like

```properties
global=(global)
```

and use Jelly

```
${%global}
```

Anyway, try it.

Reply:
You're right.